### PR TITLE
Refactoring cozy init :dizzy:

### PR DESCRIPTION
**Pull request by enguerran**
I don't know how to update #79.

And I fix as well #77 in this PR.

So @aenario and @kosssi, are we cool?

### CODE REVIEW

File: mobile/src/main.jsx [hunk 1]
```diff
@@ -39,10 +39,7 @@ document.addEventListener('DOMContentLoaded', () => {
   localforage.getItem('state').then(state => {
     if (state) {
       store.dispatch({ type: SET_STATE, state })
-      cozy.init({
-        cozyURL: state.serverUrl,
-        offline: {doctypes: ['io.cozy.files'], timer: 15}
-      })
+      init(state.serverUrl)
```
Comment (kosssi) [inline mobile/src/main.jsx]: I have always this error: https://github.com/cozy/cozy-client-js/blob/master/src/index.js#L180

Comment (kosssi) [inline mobile/src/main.jsx]: This problem is only on ios. I change it on another PR

Comment (enguerran) [inline mobile/src/main.jsx]: I will check that with `cozy-client-js-0.0.9` and will push a fix.

Comment (enguerran) [inline mobile/src/main.jsx]: I checked with @aenario:

For third party application, cozy-client must be init with a oauth block. So I add this oauth block in the cozy-helper `init` function. No need for `onRegister` callback as I explain in https://github.com/cozy/cozy-files-v3/pull/83/commits/b90e252120d709fd4da48075c6182461e94116c6.

**Comment by kosssi** [comment]
\o/

**Pull request merged**